src/dlgTriggerEditor.cpp

```diff
@@ -4048,8 +4101,8 @@ void dlgTriggerEditor::saveKey()
     }
     QString command = mpKeysMainArea->lineEdit_command->text();
     QString script = mpSourceEditor->toPlainText();
-    if( pItem )
-    {
+
+
         int triggerID = pItem->data(0, Qt::UserRole).toInt();
         TKey * pT = mpHost->getKeyUnit()->getKey( triggerID );
```

Comment:
Yeah, we should change it to a ``pKey``. This PR is about fixing colouring though